CONTRIBUTORS.rst

```diff
@@ -106,6 +106,7 @@ Listed in alphabetical order.
   Garry Cairns             `@garry-cairns`_
   Garry Polley             `@garrypolley`_
   Hamish Durkin            `@durkode`_
+  Hana Quadara             `@hanaquadara`_
```

Comment:
This is a Restructured Text link, but it's currently a broken one. We need to add the target later in the file (line ~315). It looks like this:

```
.. _@hanaquadara: https://github.com/hanaquadara
```